runtime/src/alpha/mod.rs

```diff
@@ -64,6 +64,43 @@ pub async fn start(loader: impl Loader<ProvisionerFactory> + Send + 'static) {
         }
     };
 
+    // this is handled after arg parsing to not interfere with --version above
+    if cfg!(feature = "setup-tracing") {
```

Comment:
Why this was moved to here from codegen?

Reply:
It is not needed in codegen, and it interfered with the arg parsing of runtime, which the comment states.